src/web/routes/application/journey-definitions.js

```diff
@@ -18,6 +18,7 @@ const {
 
 const APPLY = {
   name: 'apply',
+  endpoint: '/v2/claims',
```

Comment:
endpoint to me sounds like the endpoint to start the journey. claimsEndpoint might work better?

Reply:
or `postEndpoint`?

Reply:
Still sounds like an endpoint on the UI. claimServiceEndpoint or something like that. Just needs to be clear that this is the path of a down stream service at the end of the journey.

Reply:
I would rather explain this in the README (there is a story for updating this). I think `endpoint` and a search for where it is used is enough .. unless you feel really strongly about this??

Reply:
:man_shrugging: